src/views/docs.js

```diff
@@ -20,19 +20,33 @@ class Docs extends React.Component {
   }
 
   componentWillMount() {
-    const docsComponent = find(config, { route: this.props.location.pathname });
-    this.setState({
-      docs: docsComponent || { file: "" },
-      loadingError: !docsComponent ? true : false
-    });
+    this.getCurrentDocs();
   }
 
   componentDidUpdate(prevProps) {
     if (prevProps.location.pathname !== this.props.location.pathname) {
+      this.getCurrentDocs();
       window.scrollTo(0, 0);
     }
   }
 
+  getCurrentDocs() {
+    let docsComponent = { file: "" };
+    let counter = 0;
+    config.forEach((doc) => {
+      if (doc.route === this.props.location.pathname || `${doc.route}/` === this.props.location.pathname) {
+        docsComponent = doc;
+        return false;
+      }
+      counter++;
+    });
+
+    this.setState({
+      docs: docsComponent,
+      loadingError: counter === config.length
+    });
```

Comment:
@bmathews not if I have it inside of the if statement though, right? I mean I could use `componentWillReceiveProps` for this & the window scroll, would that be better?